RELEASE_NOTES.md

```diff
@@ -2,7 +2,9 @@
 
 ## Summary
 
-<!-- Here goes a general summary of what this release is about -->
+The most notable features for this release is the addition of the `PVPool` (exposed via `microgrid.pv_pool()`), which can be used to manage PV arrays as a single entity and the `EVChargerPool` (`microgrid.ev_charger_pool()`) also learning to manage power (before it could only used to retrieve data).
```

Comment:
```suggestion
The most notable features for this release is the addition of the `PVPool` (exposed via `microgrid.pv_pool()`), which can be used to manage PV arrays as a single entity and the `EVChargerPool` (`microgrid.ev_charger_pool()`) also learning to manage power (before it could be only used to retrieve data).
```

Reply:
Earlier, EV pool allowed control of individual chargers. Now that is removed and pool level control is added, using the power manager and power distributor.

Reply:
The removal was done in a separate PR, in this release cycle.  That's why EV stuff was in upgrading and not in new features.

Reply:
OK, I see, I will update.